### PR TITLE
Containerize broadway-grader

### PR DESCRIPTION
**Pull request by rod-lin**
Changes:
1. Merged utils.py to run.py
2. Passes API_HOSTNAME and API_PORT through command-line arguments
3. Added Dockerfile
4. Set chainlink workdir to "/tmp"

Linked PR: https://github.com/illinois-cs241/broadway-api/pull/47

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RUN apk add python3 git
+
+COPY src /grader
+
+RUN pip3 install -r /grader/requirements.txt
+
+RUN echo "#! /bin/sh" >> /grader/run.sh && \
+    echo "cd /grader && python3 run.py \"\$@\"" >> /grader/run.sh && \
+    chmod +x /grader/run.sh && \
+    ln -s /grader/run.sh /usr/bin/grader
```
Comment (bhuvy2) [inline Dockerfile]: Instead of creating the script like this, can you make a separate file and copy it in?

**Comment by ayushr2** [comment]
please see #13. We were avoiding `/tmp` usage.

File: Dockerfile (hunk 1 above)
Comment (rod-lin) [reply Dockerfile]: Yeah sure

**Comment by rod-lin** [comment]
Oh ok I'll switch to other directories

**Comment by ayushr2** [comment]
Can we get this up to date with all changes made to the repo?

**Comment by rod-lin** [comment]
outdated pr

**Pull request closed without merging**